index.js

```diff
@@ -8,11 +8,100 @@
 'use strict';
 
 var fs = require('fs');
+var path = require('path');
+
+/**
+ * Check if the given `filepath` exists.
+ *
+ * ```js
+ * var res = exists('package.json');
+ * console.log(res);
+ * //=> "package.json"
+ *
+ * var res = exists('fake-file.json');
+ * console.log(res)
+ * //=> false
+ * ```
+ *
+ * @name exists
+ * @param  {String} `filepath` filepath to check for.
+ * @return {String|Boolean} Returns the found filepath if it exists, otherwise returns `false`.
```

Comment:
Isn't `false` what `fs.existSync` would return? When a path is found, we return the path since `true` wouldn't be a reliable result on case-insensitive file systems, since the user wouldn't know "which path" existed.